lib/optimist/chronic.rb

```diff
@@ -0,0 +1,36 @@
+require 'chronic'
+
+
+module Optimist
+
+# Option for dates using Chronic gem.
+# Mainly for compatibility with Optimist.
+# Use of Chronic switches to United States formatted
+# dates (MM/DD/YYYY) as opposed to DD/MM/YYYY
+
+class ChronicDateOption < Option
+  register_alias :chronic_date
```

Comment:
@kbrock  I'm registering the use of `:chronic_date` here, and can include another register_alias to :natural_date - is that what you mean?